src/CSS/ListItem.css

```diff
@@ -1,9 +1,51 @@
+.purchased {
+  background-color: #ffffff;
+  color: #000000;
+  border: 1px solid #cccccc;
+}
+
 .not-purchased {
-  background-color: blue;
-  color: white;
+  background-color: #09d3ac;
```

Comment:
One thing that might be cool/useful: since we're using the same color (#09d3ac) in quite a few places, why not use a [CSS variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)? Then if/when you want to change the color of say different buttons, it's as simple as change the value of one variable rather than _every_ selector that uses the color.

Reply:
I created CSS variables for the colors I am using. Would you create CSS variables for black and white?

Reply:
@SaraSweetie if it's something you're referring to a number of times in the css files, it's a great idea. Maybe in the future you want to change to a dark gray for the black color, or a softer white instead of white.

Reply:
Exactly what @laurenmbeatty said 💯 It's all about making future changes to a project easier.